tests/framework/CodeInterfaceTests/testSaphire/sample/1/Publish/et_uq.csv

```diff
@@ -0,0 +1,5 @@
+Event Tree Sequence Uncertainty

```

Comment:
Is the binary file above used for the Interface check? If not, it can be replaced with a dummy zip (A text file renamed ```.zip```)

Reply:
It is not checked by the interface check. I put it there since I want to run that case when sapphire is present.

Reply:
Is the model sharable? Openly?

Reply:
Good question. It is a simple model, and I think it should be ok. Let me ask Ted first, and I will let you know tomorrow.